packages/contract/package.json

```diff
@@ -16,7 +16,7 @@
   "main": "index.js",
   "scripts": {
     "build": "yarn compile",
-    "compile": "sh -c \"mkdir -p ./dist\" && browserify --debug ./index.js | exorcist ./dist/truffle-contract.js.map > ./dist/truffle-contract.js && uglifyjs ./dist/truffle-contract.js -o ./dist/truffle-contract.min.js",
+    "compile": "NODE_ENV=production webpack --config ./webpack.config.js && NODE_ENV=development webpack --config ./webpack.config.js",
```

Comment:
How does this along with the webpack config make browser / node install separate versions of the truffle contract package?

Reply:
The environment variable is read by webpack I believe which alters its output.